### PR TITLE
Added COMPFILTER to documentation

### PR DESCRIPTION
**Pull request by LarsSchy**
Description of the compfilter based on original info I read, is in this users-list entry by Thomas B.

https://lists.osgeo.org/pipermail/mapserver-users/2015-October/078298.html

Fixes:  Compfilter description is missing in the COMPISTE page #677

### CODE REVIEW

File: en/mapfile/composite.txt [hunk 1]
```diff
@@ -40,10 +40,29 @@ OPACITY [integer]
...
+ :name: mapfile-composite-compfilter
+  
+COMPFILTER [string]
+    Name of the compositing filter. The primary purpose is to enable soft shadow 
+    and blurring effects, although other usages can exist or could be added in 
+    the future. Several compfilters can be placed after another to create a combined
+    effect. For example a blur and translate effect can be combined to make a 
+    soft shadow effect behind a feature.
+    
+The currently available filters are:
```
Comment (geographika) [inline en/mapfile/composite.txt:54]: The next 7 lines should be indented to match the previous paragraph, so they appear indented under the COMPFILTER section

Comment (LarsSchy) [reply en/mapfile/composite.txt:54]: That is fine that you saw that.  I was a little unsure about the indentetions and tried to follow the previous example.

**Comment by jmckenna** [comment]
@LarsSchy thanks for this.  (I also committed some minor reordering, add version info, etc, in your pull request)  Thanks again.

**Comment by jmckenna** [comment]
manually backported to branch-7-6

**Pull request merged**